libs/chain-signatures/contract/src/lib.rs

```diff
@@ -1047,6 +1072,14 @@ impl VersionedMpcContract {
         }
     }
 
+    pub fn get_tee_participants(&self) -> Vec<AccountId> {
```

Comment:
Could you add a docstring?

I find this method name ambiguous.
It can be interpreted as "the returned vector contains all `AccountId`s of participants that are running inside a TEE". But the returned `AccountId`s  may not be participants of the protocol.

How about `get_tee_accounts`?